extensions/amp-story/1.0/media-pool.js

```diff
@@ -975,58 +905,4 @@ export class MediaPool {
 
     return task.whenComplete();
   }
-
-  /**
-   * @param {!MediaPoolRoot} root
-   * @return {!MediaPool}
-   */
-  static for(root) {
-    const element = root.getElement();
-    const existingId = element[POOL_MEDIA_ELEMENT_PROPERTY_NAME];
-    const hasInstanceAllocated = existingId && instances[existingId];
-
-    if (hasInstanceAllocated) {
-      return instances[existingId];
-    }
-
-    const newId = String(nextInstanceId++);
-    element[POOL_MEDIA_ELEMENT_PROPERTY_NAME] = newId;
-    instances[newId] = new MediaPool(
-      getWin(root.getElement()),
-      root.getMaxMediaElementCounts(),
-      (element) => root.getElementDistance(element)
-    );
-
-    return instances[newId];
```

Comment:
In practice, we're scoping one MediaPool per each `amp-story`, so we now expose a much simpler public getter in the latter.